be/src/vec/exec/format/file_reader/new_plain_text_line_reader.cpp

```diff
@@ -325,13 +324,13 @@ Status NewPlainTextLineReader::read_line(const uint8_t** ptr, size_t* size, bool
         *eof = true;
         return Status::OK();
     }
-    _line_reader_ctx->refresh();
     int found_line_delimiter = 0;
     size_t offset = 0;
     bool stream_end = true;
     while (!done()) {
         // find line delimiter in current decompressed data
         uint8_t* cur_ptr = _output_buf + _output_buf_pos;
+        _line_reader_ctx->refresh();
```

Comment:
This fix may result in bad case in very wide table, aka per line is larger than read buffer size.
We previously not did so for the parse error case is low probability and for performance reason.